assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -0,0 +1,208 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+import type { Currency } from 'helpers/internationalisation/currency';
+import { updateSortCode, updateAccountNumber, updateAccountHolderName, updateAccountHolderConfirmation } from 'components/directDebit/directDebitActions';
+
+
+// ---- Types ----- //
+
+/* eslint-disable react/no-unused-prop-types */
+type PropTypes = {
+  amount: number,
+  callback: Function,
```

Comment:
Could this name be a bit more descriptive?

Reply:
I am going to look deeper into the props (including props name) in the second phase of this feature.